ruby_event_store/lib/ruby_event_store/mappers/domain_event_mapper.rb

```diff
@@ -0,0 +1,22 @@
+module RubyEventStore
+  module Mappers
+    class DomainEventMapper
```

Comment:
Btw. shouldn't we call this transformations? It does not implement Mapper API after all and that might be misleading when we already have precise meaning for mapper in RES.

Reply:
Yeah - naming is hard. Separate commit with renames shall be pushed soon.